docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -105,12 +105,12 @@ GET /_cluster/allocation/explain
 
 [source,console]
 --------------------------------------------------
-PUT /idx?master_timeout=1s&timeout=1s
+PUT /myindex?master_timeout=1s&timeout=1s
 {"settings": {"index.routing.allocation.include._name": "non_existent_node"} }
 
 GET /_cluster/allocation/explain
 {
-  "index": "idx",
+  "index": "myindex",
   "shard": 0,
   "primary": true
 }
```

Comment:
Forgot to add: You'll also need to include  a `// TEST[continued]` comment at the end of this snippet.